Kickstarter-iOS/Tests/Views/DiscoveryFiltersViewControllerTests.swift

```diff
@@ -68,6 +68,7 @@ internal final class DiscoveryFiltersViewControllerTests: TestCase {
   override func setUp() {
     super.setUp()
     AppEnvironment.pushEnvironment(mainBundle: Bundle.framework)
+    self.cache["discovery_filters_view_model_categories"] = categories
```

Comment:
@mbrandonw would you be down if I moved our loose key strings into `KSCache` enums?